nupic/encoders/multi.py

```diff
@@ -140,3 +141,66 @@ def addMultipleEncoders(self, fieldEncodings):
                 "some required constructor parameters. Parameters "
                 "that were provided are: %s" %  (encoderName, fieldParams))
           raise
+
+
+  _attrMap = {
```

Comment:
I'd just make this a constant outside the class. And you might as well move the construction of the inverted dict with this.